lib/net.c

```diff
@@ -1119,6 +1119,63 @@ start_iface(struct gatekeeper_if *iface)
 	return ret;
 }
 
+static int
+init_net_stage1(void *arg)
+{
+	struct net_config *net_conf = arg;
+	uint32_t i;
+
+	if (net_conf->gatekeeper_pktmbuf_pool == NULL) {
+		net_conf->gatekeeper_pktmbuf_pool =
```

Comment:
Given that we will now initialize the memory pools in stage 1 instead of during a call to `gatekeeper_init_network()`, we no longer need to test if `net_conf->gatekeeper_pktmbuf_pool == NULL`, that is, we can drop the test and unindent the body of the `if` statement.

Reply:
Discard this comment because we are not cleaning up if a failure occurs.